File: uniapp_flutter/lib/utils/project_submission_constants.dart
class ProjectSubmissionConstants {
  static String PROJECT_SUBMITTED_SUCCESSFULLY = "Project submitted successfully.";
  static String PROJECT_SUBMISSION_VALIDATION_ERROR = "Validation failed for project.";
  static String PROJECT_SUBMISSION_FAILED = "Project submission failed.";
  static String PROJECT_TO_SUBMIT_IN_BACKGROUND_SYNC = "Project submitted offline!";
  static String TOKEN_EXPIRY_MESSAGE = "Session Expired, Project Submitted Offline." +
      "\nPlease login again";
  static String PROJECT_SUBMITTED_OFFLINE = "Project submitted offline";
  static String PROJECT_DELETED_MESSAGE = "Project has already been deleted.";

  // Server Error Codes
  static int SUBMISSION_VALIDATION_ERROR = 420;
  static int APP_VERSION_MISMATCH_ERROR = 352;
  static int SUCCESS = 200;
  static int TOKEN_EXPIRY_ERROR = 350;
  static int PROJECT_DELETED_ERROR = 351;
  static int DEFAULT_SUBMISSION_RETRIES = 20;
  static int DEFAULT_RETRY_FREQUENCY = 7200000;
}

File: app/app/src/main/java/com/vassar/unifiedapp/utils/ProjectSubmissionConstants.java
package com.vassar.unifiedapp.utils;

import com.vassar.unifiedapp.err.UAAppErrorCodes;

public interface ProjectSubmissionConstants {
    String PROJECT_SUBMITTED_SUCCESSFULLY = "Project submitted successfully.";
    String PROJECT_SUBMISSION_VALIDATION_ERROR = "Validation failed for project.";
    String PROJECT_SUBMISSION_FAILED = "Project submission failed.";
    String PROJECT_TO_SUBMIT_IN_BACKGROUND_SYNC = "Project submitted offline!";
    String TOKEN_EXPIRY_MESSAGE = "Session Expired, Project Submitted Offline." +
            "\nPlease login again";
    String PROJECT_SUBMITTED_OFFLINE = "Project submitted offline";
    String PROJECT_DELETED_MESSAGE = "Project has already been deleted.";

    // Server Error Codes
    int SUBMISSION_VALIDATION_ERROR = 420;
    int APP_VERSION_MISMATCH_ERROR = 352;
    int SUCCESS = 200;
    int TOKEN_EXPIRY_ERROR = 350;
    int PROJECT_DELETED_ERROR = 351;
    int DEFAULT_SUBMISSION_RETRIES = 20;
    Long DEFAULT_RETRY_FREQUENCY = 7200000L;
}
